src/math/ed25519.cairo

```diff
@@ -222,6 +221,35 @@ impl PointIntoExtendedHomogeneousPoint of Into<Point, ExtendedHomogeneousPoint>
     }
 }
 
+/// Function that concats two arrays together
+/// # Arguments
+/// * `a` - First array.
+/// * `b` - Second array.
+/// # Returns
+/// * `Array<u8>` - true if the signature fits to the message and the public key, false otherwise.
+fn concat(a: @Array<u8>, b: @Array<u8>) -> Array<u8> {
```

Comment:
Weird to have this function here

Reply:
Is it alright to have dependencies between different modules of the Alexandria library? I put that there to avoid that

Reply:
If those dependency become too much of a mess, we might consider to move the most used in a common folder
But I'd try and avoid them as much as possible

Reply:
In my opinion, here it makes sense to use the ArrayTraitExt from data_structures then, there's virtually no risk of a cyclic dependency between ed25519 and array_ext

Reply:
At this point I'm fine having it but it is good to keep it in mind and monitor those once in a while 🙂